src/OpenTelemetry/Metrics/MetricStreamConfiguration.cs

```diff
@@ -10,6 +12,8 @@ public class MetricStreamConfiguration
 {
     private string? name;
 
+    private int? maxMetricPointsPerMetricStream = 2000;
```

Comment:
I think either...

a) Set this to `null` initially.
or
b) Introduce a constant and set this and the sdk default to the value. So they don't get out of sync if one happens to change 😄